@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -178,29 +262,28 @@ export namespace DisplacementFeature {
      * @param shader [[THREE.WebGLShader]] containing the vertex and fragment shaders to add the
      *                  special includes to.
      */
-    export function onBeforeCompile(displacementMaterial: DisplacementFeature, shader: any) {
-        if (displacementMaterial.displacementMap === undefined) {
+    export function onBeforeCompile(
+        displacementMaterial: DisplacementFeature,
+        shader: THREE.Shader
+    ) {
+        if (DisplacementFeature.isEnabled(displacementMaterial)) {
```

Comment:
That condition should be negated: if (**!**DisplacementFeature.isEnabled(displacementMaterial))

Reply:
Yeah, of course :), thanx for spotting.